src/TConsole.cpp

```diff
@@ -2249,7 +2249,22 @@ void TConsole::createMapper(int x, int y, int width, int height)
     }
     mpMapper->resize(width, height);
     mpMapper->move(x, y);
+
+    // Qt bug workaround: on Windows and during profile only, if the mapper widget is created
+    // it gives a height and width to mpLeftToolBar, mpRightToolBar, and mpTopToolBar for
+    // some reason. Those widgets size back down immediately after on their own (?!), however if
+    // getMainWindowSize() is called right after map create, the sizes reported will be wrong
+#if defined(Q_OS_WIN32)
+    mpLeftToolBar->setHidden(true);
```

Comment:
Are they always visible but possibly with a size of 0? Otherwise you'd have to save the state first.

Reply:
Always visible with a size zero, right. For some reason they gain dimensions right when the mapper is created and lose them in the next application cycle.